resources/lib/svt.py

```diff
@@ -333,15 +333,15 @@ def getItems(section_name, page):
 
   returned_items = []
   for video in json_data["data"]:
-    is_program = video.get("hasEpisodes", False)
+    is_program = video.get("episodic", False)
     item = {}
     item["title"] = video["programTitle"]
-    if is_program:
-      item["url"] = video["contentUrl"]
-      item["type"] = "program"
-    else:
-      item["url"] = "video/" + str(video["id"])
-      item["type"] = "video"
+    #if is_program:
+    #  item["url"] = video["contentUrl"]
+    #  item["type"] = "program"
+    #else:
+    item["url"] = "video/" + str(video["versions"][0]["id"])
+    item["type"] = "video"
```

Comment:
With this change it seems to work but i guess we should conditionally pick version somehow

Reply:
yep, this is sketchy :)